gno.land/cmd/gnoland/testdata/event_normal.txtar

```diff
@@ -0,0 +1,59 @@
+# load the package from $WORK directory
+loadpkg gno.land/r/demo/ee $WORK
+
+# start a new node
+gnoland start
```

Comment:
Do you need the `--lazy` flag? Not sure if txtar directly references the new command updates

Reply:
Don't need. Seems to be `txtar` doesn't reference new flag.